airflow/operators/email_operator.py

```diff
@@ -26,11 +28,13 @@ def __init__(
             to,
             subject,
             html_content,
+            files=[],
```

Comment:
That's a python gotcha! never instantiate in default of a method header or all functions calls will refer to the same object id. Instead, go `file=None`, and in the body of the function
`self.files = files or []`